solr/blacklight-core/conf/managed-schema

```diff
@@ -511,7 +511,10 @@
    <field name="timestamp" type="date" indexed="true" stored="true" default="NOW" multiValued="false"/>
    <!-- default, catch all search field -->
    <field name="text" type="text" indexed="true" stored="false" multiValued="true"/>
-   
+
+   <!-- search field for cancelled isbn-->
+   <field name="isbn_cancelled_t" type="text" indexed="true" stored="false" multiValued="true"/>
```

Comment:
This change isn't required. You could use the dynamic field `*_tim`  instead.  Is there an advantage you can see to explicitly defining this field?
https://github.com/ualbertalib/blacklight_solr_conf/blob/631341879705a4b86286be8effdc61eab4d90abb/solr/blacklight-core/conf/managed-schema#L558

There is a bunch of misuse of the dynamic suffixes which should be cleaned up and simplified. https://github.com/ualbertalib/blacklight_solr_conf/issues/3

Then `isbn_cancelled_t` would become `isbn_cancelled_tim` everywhere else.